egs2/iam/ocr1/run.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+train_set="train"
+valid_set="valid"
+test_set="test"
+
+asr_tag="conformer_full_vocab"
```

Comment:
We can just reply on the default `asr_tag` here for the recipe

Reply:
done!